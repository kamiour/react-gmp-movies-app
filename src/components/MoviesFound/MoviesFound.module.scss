@import 'src/scss/variables';

.moviesFound {
  font-style: normal;
  font-weight: 400;
  font-size: 20px;
  line-height: 24px;
  color: $color_white;
  padding: 24px 0 28px;
  text-align: left;

  .moviesFoundValue {
    font-weight: 600;
  }
}
