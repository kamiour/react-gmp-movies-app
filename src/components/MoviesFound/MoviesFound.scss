@import 'src/scss/variables';

.movies-found {
  font-style: normal;
  font-weight: 400;
  font-size: 20px;
  line-height: 24px;
  color: $color_white;
  padding: 24px 0 28px;
  text-align: left;

  .movies-found-value {
    font-weight: 600;
  }
}
