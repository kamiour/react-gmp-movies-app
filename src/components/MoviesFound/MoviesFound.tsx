import './MoviesFound.scss';

interface MoviesFoundProps {
  numberOfMovies: number;
}

function MoviesFound({ numberOfMovies }: MoviesFoundProps) {
  return (
    <div className="movies-found">
      <span className="movies-found-value">{numberOfMovies}</span> movies found
    </div>
  );
}

export default MoviesFound;
