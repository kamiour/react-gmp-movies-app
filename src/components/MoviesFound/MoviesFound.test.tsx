import { render } from '@testing-library/react';
import MoviesFound from './MoviesFound';

describe('MoviesFound', () => {
  it('should match MoviesFound snapshot', () => {
    const { asFragment } = render(<MoviesFound numberOfMovies={5} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
