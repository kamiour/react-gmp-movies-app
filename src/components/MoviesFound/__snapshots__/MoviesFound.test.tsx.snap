// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesFound should match MoviesFound snapshot 1`] = `
<DocumentFragment>
  <div
    class="movies-found"
  >
    <span
      class="movies-found-value"
    >
      5
    </span>
     movies found
  </div>
</DocumentFragment>
`;
